pkg/apis/workflow/v1alpha1/types.go

```diff
@@ -588,6 +591,24 @@ type HTTPArtifact struct {
 	URL string `json:"url"`
 }
 
+// FTPArtifact allows a file sto be server or loaded to a FTP server
+type FTPArtifact struct {
+	// Endpoint is the FTP server address
+	Endpoint string `json:"endpoint"`
+
+	// File to download
+	File string `json:"file"`
```

Comment:
Can we rename `Endpoint` to `Address` and `File` to `Path`? I think it will be slightly more accurate terminology and is the same terminology which the underlying SSH and GoFTP libraries are using.